src/main/java/net/fabricmc/loom/mixin/MixinMappingWriterTiny.java

```diff
@@ -46,20 +51,53 @@ public MixinMappingWriterTiny(Messager messager, Filer filer) {
 	@Override
 	public void write(String output, ObfuscationType type, IMappingConsumer.MappingSet<MappingField> fields, IMappingConsumer.MappingSet<MappingMethod> methods) {
 		if (output != null) {
-			String from = type.getKey().split(":")[0];
-			String to = type.getKey().split(":")[1];
+			String[] parts = type.getKey().split(":");
+			String from = parts[0];
+			String to = parts[1];
+
+			Map<String, Collection<String>> classesData = new TreeMap<>(); // Sorted to make 
```

Comment:
Why `TreeMap` instead of `HashMap`?

Reply:
I literally said "sorted" so that when we have tinyv2 with sorted keys lookup will be faster

Reply:
Oh, I didn't realize that's what you meant.